drivers/SPI.h

```diff
@@ -179,75 +186,85 @@ class SPI : private NonCopyable<SPI> {
+     */
     int transfer(const void *tx_buffer, int tx_length, void *rx_buffer, int rx_length, unsigned char bit_width, const event_callback_t &callback, int event);
 
-    /**
+    /** Put a transfer on the transfer queue.
      *
      * @param tx_buffer The TX buffer with data to be transfered. If NULL is passed,
-     *                  the default SPI value is sent
-     * @param tx_length The length of TX buffer in bytes
+     *                  the default SPI value is sent.
+     * @param tx_length The length of TX buffer in bytes.
      * @param rx_buffer The RX buffer which is used for received data. If NULL is passed,
-     *                  received data are ignored
-     * @param rx_length The length of RX buffer in bytes
-     * @param bit_width The buffers element width
-     * @param callback  The event callback function
-     * @param event     The logical OR of events to modify
-     * @return Zero if a transfer was added to the queue, or -1 if the queue is full
-    */
+     *                  received data are ignored.
+     * @param rx_length The length of RX buffer in bytes.
+     * @param bit_width The buffers element width in bits.
+     * @param callback  The event callback function.
+     * @param event     The event mask of events to modify.
+     *
+     * @return Operation success.
+     * @retval 0 A transfer was added to the queue.
+     * @retval -1 Transfer can't be added because queue is full.
+     */
     int queue_transfer(const void *tx_buffer, int tx_length, void *rx_buffer, int rx_length, unsigned char bit_width, const event_callback_t &callback, int event);
 
-    /** Configures a callback, spi peripheral and initiate a new transfer
+    /** Configure a callback, SPI peripheral, and initiate a new transfer.
      *
      * @param tx_buffer The TX buffer with data to be transfered. If NULL is passed,
-     *                  the default SPI value is sent
-     * @param tx_length The length of TX buffer in bytes
+     *                  the default SPI value is sent.
+     * @param tx_length The length of TX buffer in bytes.
      * @param rx_buffer The RX buffer which is used for received data. If NULL is passed,
-     *                  received data are ignored
-     * @param rx_length The length of RX buffer in bytes
-     * @param bit_width The buffers element width
-     * @param callback  The event callback function
-     * @param event     The logical OR of events to modify
-    */
+     *                  received data are ignored.
+     * @param rx_length The length of RX buffer in bytes.
+     * @param bit_width The buffers element width.
+     * @param callback  The event callback function.
+     * @param event     The event mask of events to modify.
+     */
     void start_transfer(const void *tx_buffer, int tx_length, void *rx_buffer, int rx_length, unsigned char bit_width, const event_callback_t &callback, int event);
 
+#if !defined(DOXYGEN_ONLY)
```

Comment:
It looks like this is causing the travis failure. @0xc0170 @cmonr does anyone know if DOXYGEN_ONLY is set for all travis jobs?